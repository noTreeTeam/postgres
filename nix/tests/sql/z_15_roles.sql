-- version-specific role memberships
select
    r.rolname as member,
    g.rolname as "member_of (can become)",
    m.admin_option
from
    pg_roles r
join
    pg_auth_members m on r.oid = m.member
left join
    pg_roles g on m.roleid = g.oid
order by
    r.rolname, g.rolname;
