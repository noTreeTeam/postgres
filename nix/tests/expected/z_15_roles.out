-- version-specific role memberships
select
    r.rolname as member,
    g.rolname as "member_of (can become)",
    m.admin_option
from
    pg_roles r
join
    pg_auth_members m on r.oid = m.member
left join
    pg_roles g on m.roleid = g.oid
order by
    r.rolname, g.rolname;
         member          | member_of (can become) | admin_option 
-------------------------+------------------------+--------------
 authenticator           | anon                   | f
 authenticator           | authenticated          | f
 authenticator           | service_role           | f
 pg_monitor              | pg_read_all_settings   | f
 pg_monitor              | pg_read_all_stats      | f
 pg_monitor              | pg_stat_scan_tables    | f
 pgsodium_keyholder      | pgsodium_keyiduser     | f
 pgsodium_keymaker       | pgsodium_keyholder     | f
 pgsodium_keymaker       | pgsodium_keyiduser     | f
 postgres                | anon                   | f
 postgres                | authenticated          | f
 postgres                | pg_monitor             | f
 postgres                | pg_read_all_data       | f
 postgres                | pg_signal_backend      | f
 postgres                | pgtle_admin            | f
 postgres                | service_role           | f
 supabase_read_only_user | pg_read_all_data       | f
 supabase_storage_admin  | authenticator          | f
(18 rows)

